src/UI/Button.elm

```diff
@@ -217,8 +217,11 @@ toEl cfg ((Button { click, body } _) as btn) =
 
 baseAttrs : Button msg -> List (Attribute msg)
 baseAttrs btn =
+    -- Text attributes are the same of subtitle1
     [ Font.size 16
-    , Font.center
+    , Element.spacing 8
```

Comment:
As Button is an internal component, you can use `Text.attributes cfg Text.SizeSubtitle2 Text.defaultColor`
Though, that shouldn't be necessary as we're using the text component for rendering it :thinking: